strings/redpill.json

```diff
@@ -0,0 +1,18 @@
+{
+	"redpill_questions": [
+		"What happened to the ceiling?",
+		"Why is it called the emergency shuttle if it arrives every single shift?",
```

Comment:
Can be taken out since we have crew transfer

Reply:
Crew transfer still uses the emergency shuttle